src/components/UI/DropZone/DropZone.tsx

```diff
@@ -4,14 +4,15 @@ interface DropZoneProps {
   isDragActive?: boolean;
   error?: boolean;
   children: React.ReactNode;
+  customDropZoneCss?: string;
 }
 
-export const DropZone: FunctionComponent<DropZoneProps> = ({ children, isDragActive, error }) => {
+export const DropZone: FunctionComponent<DropZoneProps> = ({ children, isDragActive, error, customDropZoneCss }) => {
```

Comment:
can we refactor this component? use this as a reference -> https://react-dropzone.js.org/#!/Styling%20Dropzone

- rename this filename as not to clash with `Dropzone` itself
- base style can apply directly to `className`
- what we wanna make use is the props passing of various `activeStyle`, `acceptStyle`, `rejectStyle`